include/PathUtil.h

```diff
@@ -48,12 +48,14 @@ namespace lmms::PathUtil
 	QString LMMS_EXPORT basePrefix(const Base base);
 	//! Check the prefix of a path and return the base it corresponds to
 	//! Defaults to Base::Absolute
-	Base LMMS_EXPORT baseLookup(const QString & path);
+	Base LMMS_EXPORT baseLookup(const QString & input);
```

Comment:
```suggestion
	Base LMMS_EXPORT baseLookup(const QString& input);
```
You missed the coding convention changes in this header file